### PR TITLE
V22F-403 Load indicator font

### PR DESCRIPTION
**Pull request by larsdekwant**

### CODE REVIEW

File: speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp [hunk 1]
```diff
@@ -468,9 +468,9 @@ void cGrBoard::DispIndicatorText(tTextData* p_data)
     if (!p_data) return;
 
     GfuiDrawString(
-        p_data->Text, normal_color_, GFUI_FONT_LARGE_C, 
-        rightAnchor * p_data->ScrPos.X,
-        TOP_ANCHOR  * p_data->ScrPos.Y);
+        rightAnchor * p_data->ScrPos.X, 
+        TOP_ANCHOR * p_data->ScrPos.Y,
```
Comment (suppergerrie2) [inline speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp:472]:
```suggestion
        rightAnchor* p_data->ScrPos.X, 
        TOP_ANCHOR* p_data->ScrPos.Y, 
```

**Review by suppergerrie2 — CHANGES_REQUESTED** [review]

File: speed-dreams/source-2.2.3/src/libs/tgfclient/guiscrollist.cpp [hunk 2]
```diff
@@ -651,7 +651,7 @@ gfuiDrawScrollist(tGfuiObject *obj)
 				break;
 			}
 			y -= h;
-			gfuiDrawString(x+5, y, scrollist->font, elt->label);
+            gfuiDrawString(x+5, y, scrollist->font, elt->label);
```
Comment (shoekzema) [inline speed-dreams/source-2.2.3/src/libs/tgfclient/guiscrollist.cpp:654]: Dont push a changed SD file with only some spaces as difference

File: speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h [hunk 3]
```diff
@@ -456,6 +456,9 @@ TGFCLIENT_API int GfuiVisibilitySet(void* scr, int id, int visible);
 TGFCLIENT_API int GfuiEnable(void* scr, int id, int flag);
 TGFCLIENT_API void GfuiUnSelectCurrent(void);
 
+// SIMULATED DRIVING ASSISTANCE: expose the GfuiFontClass to the API
+class TGFCLIENT_API GfuiFontClass;
```
Comment (shoekzema) [inline speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h:460]: add to CHANGES.txt

File: speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h [hunk 4]
```diff
@@ -476,6 +479,9 @@ TGFCLIENT_API void GfuiUnSelectCurrent(void);
 
 TGFCLIENT_API int  GfuiFontHeight(int font);
 TGFCLIENT_API int  GfuiFontWidth(int font, const char* text);
+
+// SIMULATED DRIVING ASSISTANCE: expose method to API.
+TGFCLIENT_API void GfuiDrawString(int p_x, int p_y, const char* p_string, GfuiFontClass* p_font, float* p_color);
 TGFCLIENT_API void GfuiDrawString(const char* text, float* fgColor, int font,
                                   int x, int y, int width = 0, int hAlign = GFUI_ALIGN_HL);
```
Comment (shoekzema) [inline speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h:486]: add new GfuiDrawString function to CHANGES.txt

**Review by shoekzema — CHANGES_REQUESTED** [review]

File: speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h (hunk 4 above)
Comment (larsdekwant) [reply speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h:486]: Already in there (see last commit before the PR opened for review)

File: speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h (hunk 3 above)
Comment (larsdekwant) [reply speed-dreams/source-2.2.3/src/libs/tgfclient/tgfclient.h:460]: Already in there (see last commit before the PR opened for review)

File: speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp (hunk 1 above)
Comment (larsdekwant) [reply speed-dreams/source-2.2.3/src/modules/graphic/ssggraph/grboard.cpp:472]: These are not pointers and I will keep whitespace between arithmetic operations

File: speed-dreams/source-2.2.3/src/libs/tgfclient/guiscrollist.cpp (hunk 2 above)
Comment (larsdekwant) [reply speed-dreams/source-2.2.3/src/libs/tgfclient/guiscrollist.cpp:654]: Oh Visual Studio automatically did this when I renamed this variable at first (will fix it now)

**Review by shoekzema — APPROVED** [review]

**Review by suppergerrie2 — APPROVED** [review]

**Pull request merged**